input/write_reactors.py

```diff
@@ -5,7 +5,7 @@
 
 # tells command format if input is invalid
 if len(sys.argv) < 6:
-    print('Usage: python write_reactors.py [csv] [reactor_template] [region_template]\
+    print('Usage: python write_reactors.py [csv] [reactor_template] [deployinst_template]\
```

Comment:
Here is the output of flake8 (please consider running this before submitting a PR). 

```
huff@lise:~/repos/transition-scenarios/input(deployinst) $ flake8 write_reactors.py 
write_reactors.py:8:80: E501 line too long (80 > 79 characters)
write_reactors.py:45:56: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:45:58: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:47:72: W291 trailing whitespace
write_reactors.py:52:80: E501 line too long (87 > 79 characters)
write_reactors.py:53:73: E231 missing whitespace after ','
write_reactors.py:53:80: E501 line too long (82 > 79 characters)
write_reactors.py:54:80: E501 line too long (93 > 79 characters)
write_reactors.py:55:80: E501 line too long (80 > 79 characters)
write_reactors.py:89:1: W293 blank line contains whitespace
write_reactors.py:92:14: E225 missing whitespace around operator
write_reactors.py:104:18: E225 missing whitespace around operator
write_reactors.py:118:1: W293 blank line contains whitespace
write_reactors.py:120:1: W293 blank line contains whitespace
write_reactors.py:166:80: E501 line too long (89 > 79 characters)
write_reactors.py:202:38: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:202:40: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:202:62: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:202:64: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:247:78: E225 missing whitespace around operator
write_reactors.py:247:80: E501 line too long (91 > 79 characters)
write_reactors.py:248:66: E225 missing whitespace around operator
write_reactors.py:249:52: E225 missing whitespace around operator
write_reactors.py:250:62: E225 missing whitespace around operator
write_reactors.py:251:1: W293 blank line contains whitespace
write_reactors.py:252:48: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:252:50: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:253:49: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:253:51: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:254:45: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:254:47: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:255:47: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:255:49: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:260:5: E303 too many blank lines (2)
write_reactors.py:266:80: E501 line too long (84 > 79 characters)
write_reactors.py:267:59: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:267:61: E251 unexpected spaces around keyword / parameter equals
write_reactors.py:311:5: E265 block comment should start with '# '
write_reactors.py:315:5: E303 too many blank lines (2)
write_reactors.py:323:80: E501 line too long (98 > 79 characters)
write_reactors.py:325:36: W291 trailing whitespace
write_reactors.py:328:80: E501 line too long (86 > 79 characters)
write_reactors.py:329:80: E501 line too long (96 > 79 characters)
write_reactors.py:330:80: E501 line too long (81 > 79 characters)
```